openl3/cli.py

```diff
@@ -124,21 +171,29 @@ def parse_args(args):
     parser.add_argument('--input-repr', '-i', default='mel256',
                         choices=['linear', 'mel128', 'mel256'],
                         help='String specifying the time-frequency input '
-                             'representation for the embedding model.')
+                             'representation for the audio embedding model.')
 
     parser.add_argument('--content-type', '-c', default='music',
                         choices=['music', 'env'],
                         help='Content type used to train embedding model.')
 
-    parser.add_argument('--embedding-size', '-s', type=int, default=6144,
-                        help='Embedding dimensionality.')
+    parser.add_argument('--audio-embedding-size', '-as', type=int, default=6144,
+                        choices=[6144, 512],
+                        help='Audio embedding dimensionality.')
 
-    parser.add_argument('--no-centering', '-n', action='store_true', default=False,
-                        help='Do not pad signal; timestamps will correspond to '
+    parser.add_argument('--no-audio-centering', '-n', action='store_true',
+                        default=False,
+                        help='Used for audio embeddings. Do not pad signal; '
+                             'timestamps will correspond to '
                              'the beginning of each analysis window.')
 
-    parser.add_argument('--hop-size', '-t', type=positive_float, default=0.1,
-                        help='Hop size in seconds for processing audio files.')
+    parser.add_argument('--audio-hop-size', '-t', type=positive_float, default=0.1,
+                        help='Used for audio embeddings. '
+                             'Hop size in seconds for processing audio files.')
```

Comment:
As before, does this impact audio embeddings extracted from video files? Since audio embeddings can be extracted either just from audio files or from video files, we need to be very clear in the documentation about which parameters affect what.

Reply:
Currently this effects all audio processed, including that from videos. But yes, we should clarify this in the documentation.

Reply:
👍